Seems like a simple tolerance thing:
```
[834](https://github.com/scipy/scipy-release/actions/runs/32099350455/job/95596733327?pr=33#step:9:4848)
    FAILED stats/tests/test_stats.py::TestPearsonr::test_different_dimensionality[numpy] - AssertionError: 
    Arrays are not equal
    
    Mismatched elements: 2 / 3 (66.7%)
    Mismatch at indices:
     [0]: 0.23665652031816728 (ACTUAL), 0.2366565203181673 (DESIRED)
     [2]: -0.26031843957123957 (ACTUAL), -0.2603184395712396 (DESIRED)
    Max absolute difference among violations: 5.55111512e-17
    Max relative difference among violations: 2.13243254e-16
     ACTUAL: array([ 0.236657,  0.243693, -0.260318])
     DESIRED: array([ 0.236657,  0.243693, -0.260318])
```

It seems almost impossible to teach people to not compare floating point arrays for exact equality. A bump to rtol=3e-16 should do it.

```
    FAILED stats/tests/test_morestats.py::TestBoxcoxNormmax::test_nan_policy_all - AssertionError: 
    Not equal to tolerance rtol=1e-07, atol=0
    
    Mismatched elements: 1 / 2 (50%)
    Mismatch at index:
     [0]: 0.3071858363151042 (ACTUAL), 0.3071858912803311 (DESIRED)
    Max absolute difference among violations: 5.49652269e-08
    Max relative difference among violations: 1.78931482e-07
     ACTUAL: array([0.307186, 0.267629])
     DESIRED: array([0.307186, 0.267629])
```

This one seems new. Also a small bump needed probably, either unconditional or platform-specific to 2e-7.

--- a/.github/workflows/wheels.yml
+++ b/.github/workflows/wheels.yml
@@ -62,7 +62,7 @@ jobs:
         - [macos-14, macosx, arm64, accelerate, "14.0"]
         - [windows-2025, win, AMD64, "", ""]
         - [windows-11-arm, win, ARM64, "", ""]
-        python: ["cp312", "cp313", "cp314", "cp314t"]
+        python: ["cp312", "cp313", "cp314", "cp314t", "cp315", "cp315t"]
 
     steps:
       - name: Checkout scipy-release
@@ -130,7 +130,7 @@ jobs:
           echo "$CIBW_ENV" >> "$GITHUB_ENV"
 
       - name: Build wheels
-        uses: pypa/cibuildwheel@4726cd35bb13f7bde50cf2761f2499ac7b3aa32c # v4.1.1
+        uses: pypa/cibuildwheel@1828c10ab37f080699c7b81cea34097c684a7074 # v4.2.0
         env:
           CIBW_BUILD: ${{ matrix.python }}-${{ matrix.buildplat[1] }}*
           CIBW_ARCHS: ${{ matrix.buildplat[2] }}

